test/ggsn_SUITE.erl

```diff
@@ -2017,7 +2017,7 @@ simple_ocs(Config) ->
 	     #measurement_method{volum = 1, durat = 1},
 	 reporting_triggers =>
 	     #reporting_triggers{
-		linked_usage_reporting = 1,
+		linked_usage_reporting = 0,
```

Comment:
By default `linked_usage_reporting` has value `0`, I suppose need remove `linked_usage_reporting` from this place. The same in other places in `*_SUITE.erl`.

Reply:
updated, thanks